packages/tooltip/README.md

```diff
@@ -0,0 +1,77 @@
+# Tooltip
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/tooltip.svg)
+
+## Example
+
+<!-- will update once component is finalized -->
+
+## Properties
+
+### open
+
+**Type:** `boolean`
+
+**Default:** `false`
+
+Controls the component, and determines whether or not the `Tooltip` will appear open or closed.
+
+### setOpen
+
+**Type:** `function`
+
+If controlling the component, pass state handling function to setOpen prop. This will keep the consuming application's state in-sync with LeafyGreen's state, while the `Tooltip` component responds to events such as backdrop clicks and a user pressing the Escape key.
+
+### shouldClose
+
+**Type:** `function`
+
+**Default:** `() => true`
+
+Callback to determine whether or not `Tooltip` should close when user tries to close it.
+
+### align
+
+**Type:** `top`, `bottom`, `left`, `right`
+
+**Default:** `top`
+
+Determines the alignment of the `Tooltip` component relative to the element passed to the `trigger` prop.
+
+### justify
+
+**Type:** `start`, `middle`, `end`
+
+**Default:** `start`
+
+Determines the justification of the `Tooltip` component (based on the alignment) relative tto the element passed to the `trigger` prop.
+
+### trigger
+
+**Type:** `HTMLElement` or `ReactNode`
+
+An `HTMLElement` or `ReactNode` against which the `Tooltip` will be positioned, and what will be used to trigger the opening and closing of the `Tooltip` component.
+
+### triggerEvent
+
+**Type:** `hover` or `click`
+
+**Default:** `hover`
+
+Event that triggers opening/closing of `Tooltip` component
+
+### variant
+
+**Type:** `light` or `dark`
+
+**Default:** `light`
+
+Determines color of `Tooltip` component
+
+### id
+
+**Type:** `string`
+
+id applied to `Tooltip` component
```

Comment:
Either "ID" or `id` in code tags so that this is easier to read correctly.

Reply:
Yep makes sense